### PR TITLE
add-dbt-cloud-postman-collection-blog

### PR DESCRIPTION
**Pull request by johnblust**
## Description & motivation
Adds Matt Winkler's announcement post for the dbt Cloud API postman collection.
Adds an author profile for Matt Winkler.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by jasnonaz** [comment]
This is awesome!! Can't wait to get this out into the world. As I'm reviewing, one thing I'm thinking is that we should have a short (one to two paragraph) walkthrough on what exactly a postman collection is and how to use it - just something to give some context to folks that might be interested in this but don't have full context. @matt-winkler would you be able to add that into the draft and then we can get it folded into the post?

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: "Introducing the dbt Cloud API Postman Collection: a tool to help you scale your account management"
+description: "Discover powerful dbt Cloud endpoints with this postman collection put together by Solution Architect Matt Winkler."
+slug: dbt-cloud-api-postman-collection-announcement
+
+authors: [matt_winkler]
+
+tags: [analytics craft]
```
Comment (jasnonaz) [inline website/blog/2022-04-19-dbt-cloud-postman-collection.md]: 'dbt tutorials' might be a better fit for this one

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+We usually advise our customers to leverage API requests to automate these typically manual tasks, and you should, too!
+
+## Closing Thoughts
+
+Beyond the day-to-day process of managing their dbt Cloud accounts, many organizations benefit from being able to quickly replicate environments as they experiment with new dbt features and development patterns. Governance is another benefit of managing infrastructure in code, as resource definitions can be version controlled and inspected by the relevant teams before changes go live. These concepts are high on our minds as we look to provide additional capabilities to our customers in their use of dbt Cloud. Happy APIing!
+
+*Below this you’ll find a series of example requests - use these to guide you or [check out the Postman Collection](https://dbtlabs.postman.co/workspace/Team-Workspace~520c7ac4-3895-4779-8bc3-9a11b5287c1c/request/12491709-23cd2368-aa58-4c9a-8f2d-e8d56abb6b1dlinklink) to try it out yourself.*
+
+## Examples of how to use the Postman Collection
```
Comment (jasnonaz) [inline website/blog/2022-04-19-dbt-cloud-postman-collection.md]: Should we add 'Appendix:' here - feels kind of odd to have closing thoughts then another super long section - think the appendix callout would clear it up

**Comment by matt-winkler** [comment]
@johnblust I updated the links section with the options! Looks like things are working from testing locally, but please give this a quick smell check!

**Comment by johnblust** [comment]
Think we're good to go for a second review @jasnonaz !

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+## Closing Thoughts
+
+Beyond the day-to-day process of managing their dbt Cloud accounts, many organizations benefit from being able to quickly replicate environments as they experiment with new dbt features and development patterns. Governance is another benefit of managing infrastructure in code, as resource definitions can be version controlled and inspected by the relevant teams before changes go live. These concepts are high on our minds as we look to provide additional capabilities to our customers in their use of dbt Cloud. Happy APIing!
+
+*Below this you’ll find a series of example requests - use these to guide you or [check out the Postman Collection](https://dbtlabs.postman.co/workspace/Team-Workspace~520c7ac4-3895-4779-8bc3-9a11b5287c1c/request/12491709-23cd2368-aa58-4c9a-8f2d-e8d56abb6b1dlinklink) to try it out yourself.*
+
+## Appendix 
+
+###Examples of how to use the Postman Collection
```
Comment (KiraFuruichi) [inline website/blog/2022-04-19-dbt-cloud-postman-collection.md]:
```suggestion
### Examples of how to use the Postman Collection
```
Need an additional space between last # and 'E'

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+When first learning dbt Cloud and operating at a smaller scale, most end users tend to favor manual, GUI based workflows for creating and managing their account infrastructure. Many of our customers start off this way, but they usually end up running into these bottlenecks as the number of jobs and projects multiplies. Here are some examples of the kind of bottlenecks I hear our growing customers talk about.
+
+- “I find myself wasting time manually clicking through the UI to manage infrastructure, especially after deploying more than a handful of projects and environments.”
+- “I don’t have peace of mind that my production environment is configured exactly as intended. If a new team member deletes or changes something, I have to run a fire drill to fix it.”
+- “My organization requires me to run any environmental configuration changes through code review and version control.”
+
+We usually advise our customers to leverage API requests to automate these typically manual tasks, and you should, too!
+
+## Closing Thoughts
```
Comment (KiraFuruichi) [inline website/blog/2022-04-19-dbt-cloud-postman-collection.md:42]: I wonder if we can reword this from 'Closing Thoughts' to something else? Closing thoughts makes me think the content would be over, but it digs into an example after that. Or maybe moving the current 'Closing Thoughts' to the end? Also @johnblust can correct me on capitalization, but to be consistent with other headers, should maybe be 'Closing thoughts' with second word lowercased. Lmk what people think!

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+Our example contains the following elements:
+
+![This diagram depicts how an API request works. The diagram depicts a Dev project and a Prod project. The Dev project contains two environments: a development and deployment environment. The deployment project contains a job. The diagram shows the deployment environment that contains the job being copied over to the Prod Project via API request.](/img/blog/2022-04-19-dbt-cloud-postman-collection/example-cloud-environment.png)
+
+### Setup
+
+Create a “development” and a “production” project in dbt Cloud using the UI
+Set up a [data warehouse connection](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database) and [repository connection](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories)
+
+#### Pull the environment from the development project
```
Comment (KiraFuruichi) [inline website/blog/2022-04-19-dbt-cloud-postman-collection.md:77]: The H4s here for 'environment' changes between capitalization of the 'E' in this one and the following one. (Same for the H4s involving Jobs.) Is this on purpose or do we want to use a consistent one across the H4s here?

**Comment by KiraFuruichi** [comment]
Super cool and practical post! Added a few comments/suggestions on structure and capitalization, but think it looks good!

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md (hunk 5 above)
Comment (johnblust) [reply website/blog/2022-04-19-dbt-cloud-postman-collection.md:77]: Good catch!

File: website/blog/2022-04-19-dbt-cloud-postman-collection.md (hunk 4 above)
Comment (johnblust) [reply website/blog/2022-04-19-dbt-cloud-postman-collection.md:42]: Hmm that's a valid point. I'll see if I can come up with a better name

**Review by jasnonaz — APPROVED** [review]
Looks good!

**Pull request merged**